stdlib/ballerina-websub/src/main/java/org/ballerinalang/net/websub/WebSubHttpService.java

```diff
@@ -47,7 +47,7 @@ protected WebSubHttpService(Service service) {
     }
 
     private static Annotation getWebSubSubscriberServiceConfigAnnotation(Service service) {
-        return getServiceConfigAnnotation(service, WebSubSubscriberConstants.WEBSUB_PACKAGE_PATH,
+        return getServiceConfigAnnotation(service, WebSubSubscriberConstants.WEBSUB_PACKAGE,
```

Comment:
Shall we static import these constants? Will improve readability

Reply:
Fixed with 62b3504